### ISSUE TITLE
vroom docker not responding to large custom matrix request

### ISSUE THREAD

**Issue by ibrahimsk1** [VROOM-Project/vroom#529]
vroom docker works when i send a 5 job request with custom matrix but when i send 250 jobs with custom matrix then i dont get respond.when i try with postman for small request i get succesful response and for large one i get socket hang up error. I changed request timeout to  never timeout. I guess problem is custom matrix because for 250 jobs its about 62.500 lines. How can i solve this problem ?Does anyone have an idea? Thanks

**Comment by nilsnolde** [comment]
What does the docker log say? E.g. HTTP status code? Which router are you using?

I didn't quite understand though: And are you saying that with docker it doesn't work but without docker it works? 250 jobs shouldn't be an issue in itself (I think).

**Comment by ibrahimsk1** [comment]
i used this command to build docker image.
docker run -dt --name vroom --net host -v $PWD/conf:/conf -e VROOM_ROUTER=osrm vroomvrp/vroom-docker:v1.10.0

5 job request work with docker and 250 job request doesnt work with docker.Both request is send to same image from postman

. Docker logs doesnt say anything there are no logs for 250 job request and i get could not get response - socket hang up error. I have logs from 5 job request and get 200 success or 400 bad request response.

So i changed body json data size to find which length is accepted and find that maximum 150kb json is accepted for more i get same 'could not get response' error. 

I use 3000 port at remote linux server and made my request from postman. So i thought maybe it is about some data size restriction of linux server and i decided to try with curl from server.

i created 2 json file. first one ise example2.json from vroom docs and second is file1.json is my data. I added 2 files to this comment.

[file1.txt](https://github.com/VROOM-Project/vroom/files/6642121/file1.txt)
[example2.txt](https://github.com/VROOM-Project/vroom/files/6642122/example2.txt)

curl --header "Content-Type:application/json" -d @example2.json  http://localhost:3000 works well and returns result
curl --header "Content-Type:application/json" -d @file1.json  http://localhost:3000  doesnt work and returns 
"curl: (52) Empty reply from server"

**Comment by jcoupey** [comment]
I don't think this is related to the core codebase since I'm getting what looks like the expected response using a simple `vroom` build (no Docker):

```bash
$ vroom -i file1.txt | jq .summary
{
  "cost": 69321,
  "unassigned": 0,
  "service": 0,
  "duration": 69321,
  "waiting_time": 0,
  "priority": 0,
  "violations": [],
  "computing_times": {
    "loading": 1,
    "solving": 7350
  }
}
```
A timeout problem sounds unlikely too since the solution is computed in around 7 seconds.

Anyway, moving this issue to the `vroom-docker` repo.
